sdk/azurestackhci/arm-azurestackhci/src/operationsInterfaces/galleryImagesOperations.ts

```diff
@@ -1,149 +0,0 @@
-/*
- * Copyright (c) Microsoft Corporation.
- * Licensed under the MIT License.
- *
- * Code generated by Microsoft (R) AutoRest Code Generator.
- * Changes may cause incorrect behavior and will be lost if the code is regenerated.
- */
-
-import { PagedAsyncIterableIterator } from "@azure/core-paging";
-import { SimplePollerLike, OperationState } from "@azure/core-lro";
-import {
-  GalleryImages,
-  GalleryImagesListOptionalParams,
-  GalleryImagesListAllOptionalParams,
-  GalleryImagesGetOptionalParams,
-  GalleryImagesGetResponse,
-  GalleryImagesCreateOrUpdateOptionalParams,
-  GalleryImagesCreateOrUpdateResponse,
-  GalleryImagesDeleteOptionalParams,
-  GalleryImagesDeleteResponse,
-  GalleryImagesUpdateRequest,
-  GalleryImagesUpdateOptionalParams,
-  GalleryImagesUpdateResponse
-} from "../models";
-
-/// <reference lib="esnext.asynciterable" />
-/** Interface representing a GalleryImagesOperations. */
-export interface GalleryImagesOperations {
-  /**
```

Comment:
i didn't realize this change in changelog, do you know why?

Reply:
this operation group is introduced in last preview release